src/index.html

```diff
@@ -3,10 +3,418 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com" >
+    <link
+     rel="preconnect"
+     href="https://fonts.gstatic.com"
+     crossorigin="#"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+       href="#home"
+       class="logo-link"
+      >
+        <img
+         src="./images/logo.png"
+         alt="logo"
+         class="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+             href="#"
```

Comment:
for more readable
do it in all cases
```suggestion
         class="logo"
        >
      </a>
      
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#" class="nav__link is-active">Apple</a>
          </li>
          
          <li class="nav__item">
            <a href="#" class="nav__link">Samsung</a>
          </li>
          
          <li class="nav__item">
            <a href="#" class="nav__link">Smartphones</a>
          </li>
          
          <li class="nav__item">
            <a
             href="#"
```